### ISSUE TITLE
Add tests for multivariate distributions

### ISSUE THREAD

**Issue by fonnesbeck** [pymc-devs/pymc3#314]
As far as I can tell, there are none.

**Comment by fonnesbeck** [comment]
One issue that worries me is the lack of a sum-to-one constraint on the Multinomial probabilities:

```
def logp(x):
    # only defined for sum(p) == 1
    return bound(
        factln(n) + sum(x * log(p) - factln(x)),
        n > 0,
        0 <= x, x <= n)
```

Though the constraint is stated in the comment, there appears to be no enforcement of this constraint. The Dirichlet seems similarly unconstrained.

**Comment by twiecki** [comment]
Wasn't the dirichlet constraint problem fixed?

The lack of tests seems duplicate with #278. Close?

**Comment by jsalvatier** [comment]
I'm working on this.

**Comment by twiecki** [comment]
Fixed in #390.
